frontend/src/components/_common/InputGroup/Message/Message.tsx

```diff
@@ -0,0 +1,28 @@
+import { css } from 'styled-components';
+
+import { InputStatus } from '@/components/_common/InputGroup/Input.type';
+
+import { theme } from '@/styles/theme';
+
+import * as S from './Message.styles';
+
+interface MessageProps {
+  $css?: ReturnType<typeof css>;
+  status?: InputStatus;
+  fontSize?: string;
+}
+
+const Message = ({
+  status = 'DEFAULT',
+  fontSize = theme.fontSize.sm,
+  $css,
+  children,
```

Comment:
```suggestion
  $css,
  status = 'DEFAULT',
  fontSize = theme.fontSize.sm,
  children,
```

`props` 와 정렬 순서가 같았으면 좋겠어요!